btrack/_localization.py

```diff
@@ -191,7 +191,7 @@ def segmentation_to_objects(
 
     # we need to remove 'label' since this is a protected keyword for btrack
     # objects
-    if "label" in properties:
+    if isinstance(properties, tuple) and "label" in properties:
```

Comment:
`properties` is incrorrectly typed in the signature - it's using an emtpy tuple for a null value, so if you convert it's typing to not be optional you probably won't need to add this `isinstance` call.

Reply:
I know that it is incorrect. That's actually what prompted me to go down this rabbit hole. IMO it is actually the correct type i.e. `Optional[Tuple[str]]` but it should default to `None` rather than `()`.

The problem comes later on (which is why I haven't included it for now) is when `properties` is converted to a `set` and then back to a `tuple` https://github.com/quantumjot/BayesianTracker/pull/128/files#r869069572

Reply:
For now this "fixes" `mypy`. And I think you'd have to assert that it is `not None` instead i.e. `if properties is not None and "label" in properties`

Reply:
Without this fix:
![image](https://user-images.githubusercontent.com/15052188/167856508-b2e2a105-1524-461f-81fd-b77577a06bf8.png)

With this fix, no warning
![image](https://user-images.githubusercontent.com/15052188/167856847-7474604e-47d6-4cb0-b9f7-b145a44cc6dd.png)

Change default to `None`, warning returned
![image](https://user-images.githubusercontent.com/15052188/167857137-23db8a4e-71a4-4556-99a6-e0a669162a6e.png)

Default still `None`, warning fixed. `isinstance(properties,tuple)` would work too of course
![image](https://user-images.githubusercontent.com/15052188/167857283-f315ae59-364e-4055-9737-13f5e758ba9d.png)

I've done plenty of these before, hence I know 😆